resources/feature.rb

```diff
@@ -63,20 +64,22 @@ def run_default_provider(desired_action)
         feature_name new_resource.feature_name
         source new_resource.source if new_resource.source
         all new_resource.all
+        timout new_resource.timeout
       end
     when :windows_feature_servermanagercmd
       windows_feature_servermanagercmd new_resource.name do
         action desired_action
         feature_name new_resource.feature_name
-        source new_resource.source if new_resource.source
```

Comment:
?

Reply:
Servermanagercmd.exe doesn't actually support defining an alternate source like DISM and Powershell do

Reply:
I should read the PR before commenting. Oops